content/en/docs/started/workstation/minikf-aws.md

```diff
@@ -0,0 +1,191 @@
+   impossible to train ML models.
+
+   <img src="/docs/images/minikf-aws/minikf-aws-instance-type.png"
+       alt="Select Intance Type"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Choose an existing **VPC** or create a new one. If you create a new one, make
+   sure you click on the **refresh** icon to update the contents of the list, so
+   that your new VPC appears in it.
+
+   <img src="/docs/images/minikf-aws/minikf-aws-security-settings-vpc.png"
+       alt="Select a VP or create a new one"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Choose an existing **Subnet** or create a new one. If you create a new one,
+   make sure you click on the **refresh** icon to update the contents of the
+   list, so that your new subnet appears in it.
+
+   <img src="/docs/images/minikf-aws/minikf-aws-security-settings-subnet.png"
+       alt="Select a Subnet or create a new one"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. We strongly recommend to create a new security group. Otherwise you may not
+   be able to access your MiniKF. To do so, click on **Create New Based On
+   Seller Settings**.
+   
+   ---
+   **NOTE**
+   
+   If you need to use an existing security group, **make sure it covers
+   the ports mentioned in the usage instructions** (see at the top of this
+   page).
+
+   ---
+   
+   <img src="/docs/images/minikf-aws/minikf-aws-security-group.png"
+       alt="Create a new security group"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Choose an existing **Key Pair** or create a new one. If you create a new one,
+   make sure you click on the **refresh** icon to update the contents of the
+   list, so that your new key pair appears in it.
+   
+   <img src="/docs/images/minikf-aws/minikf-aws-security-settings-key-pair.png"
+       alt="Select a Key Pair or create a new one"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Click **Launch**. You will then see a message informing you about the
+   successful installation. The message contains a link to view the instance on
+   the EC2 Console. Click on it.
+
+   <img src="/docs/images/minikf-aws/minikf-aws-successful-deployment.png"
+       alt="Successful deployment message"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. You should now see a screen like this. Click on the **Instance ID** of the
+   created instance
+
+   <img src="/docs/images/minikf-aws/minikf-aws-ec2-console.png"
+       alt="Instance on EC2 Console"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. You will then see this page. Click **Connect** to connect to the instance
+   using EC2 Instance Connect
+
+   ---
+   **NOTE**
+
+   This will only work if you have created a security group based on seller
+   settings, as in step 10, or you have configured a security group based on the
+   usage instructions. Otherwise, you need to SSH to the instance manually, see:
```

Comment:
```suggestion
   **Note**: This will only work if you have created a security group based on seller
   settings, as in step 10, or you have configured a security group based on the
   usage instructions. Otherwise, you need to SSH to the instance manually, see:
```